xash-exec

```diff
@@ -0,0 +1,7 @@
+#!/usr/sgug/bin/bash
```

Comment:
It seems to be IRIX specific?

Could you move it to engine/platform/irix then?